lib/pg_saurus/connection_adapters/postgresql_adapter/foreign_key_methods.rb

```diff
@@ -34,7 +34,7 @@ def add_foreign_key(from_table, to_table, options = {})
           "  Use :exclude_index => true when adding the foreign key."
       end
 
-      super from_table, to_table, options
+      super from_table, to_table, **options
```

Comment:
Corresponding method in:
- Rails 6.0.0: [`ActiveRecord::ConnectionAdapters::SchemaStatements#add_foreign_key`](https://github.com/rails/rails/blob/v6.0.0/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L991)
  -  Note that the method signature has `options` as a positional argument. In this case, that is fine because we are splatting the keywords when calling the super method (see example set 3 in the demo)
- Rails 6.1.0: [`ActiveRecord::ConnectionAdapters::SchemaStatements#add_foreign_key`](https://github.com/rails/rails/blob/v6.1.0/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L1059)